composition/src/resolvability-graph/graph.ts

```diff
@@ -298,9 +298,12 @@ export class Graph {
         `resDataByRelativeOriginPath`,
       );
       const fullPath = `${pathFromRoot}${unresolvableEntityPath}`;
-      const rootResData = getOrThrowError(walker.resDataByPath, fullPath, `rootFieldWalker.resDataByPath`);
-      entityResData.addData(rootResData);
-      rootResData.addData(entityResData);
+
```

Comment:
Please add the following comment
`// The path may not exist from the root walker due to nested entities.`